pkg/apis/kops/validation/validation.go

```diff
@@ -794,32 +800,52 @@ func validateNetworkingGCE(c *kops.ClusterSpec, v *kops.GCENetworkingSpec, fldPa
 	return allErrs
 }
 
-func validateAdditionalPolicy(role string, policy string, fldPath *field.Path) field.ErrorList {
-	errs := field.ErrorList{}
+func validateAdditionalPolicies(role string, policy string, fldPath *field.Path) field.ErrorList {
```

Comment:
It's validating a singular policy, so the original name is correct.

Reply:
Wanted to name it as the field it validates, but can change back.

Reply:
Done.